src/styles/card.scss

```diff
@@ -0,0 +1,121 @@
+.card {
+  margin: auto;
+  width: 200px;
+  padding: 16px;
+  border-radius: 5px;
+  border: 1px solid $color-borrer-white;
+  background: $color-bg-white;
+  font-size: 0;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 16px 3px 0;
+  }
+
+  &__title {
+    color: $font-color-bl;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 0 0;
+  }
+
+  &__code {
+    color: $font-color-gr;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 4px 0 0;
+  }
+
+  &__review-container {
+    display: flex;
+    justify-content: space-between;
+    margin: 16px 0 0;
+  }
+
+  &__review-count {
+    display: flex;
+    align-items: center;
+    color: $font-color-bl;
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 0 0;
+  }
+
+  &__price-label {
+    color: $font-color-gr;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    color: $font-color-bl;
+    text-align: right;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &:hover .card__title {
+    color: $color-title-hover;
+  }
+
+  &__button {
+    display: block;
+    padding: 0;
+    width: 166px;
+    height: 40px;
+    line-height: 40px;
+    border-radius: 5px;
+    background: $color-button-blue;
+    text-transform: uppercase;
+    color: $color-button-white;
+    text-align: center;
+    font-size: 14px;
+    font-weight: 700;
+    margin: 16px 0 0;
+    text-decoration: none;
+    border: 1px solid transparent;
+    transition: all $time-changes;
```

Comment:
Not critical, but it's a good tone to specify transition properties
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x